h2o-docs/src/product/data-science/glm.rst

```diff
@@ -1740,7 +1741,47 @@ Regress :math:`z_{i}` on the predictors :math:`x_{i}` using the weights :math:`w
 
 This process is repeated until the estimates :math:`\hat{\beta}` change by less than the specified amount.
 
-**Cost of computation**
+**Tweedie Likelihood Calculation**
+
+You only calculate Tweedie likelihood when you estimate the Tweedie variance power or the Tweedie variance power with dispersion. The calculation in this section is used to estimate the full log likelihood. When you fix the Tweedie variance power, you will use a simpler formula (unless you are estimating dispersion). When fixing the Tweedie variance power for dispersion estimation, you use the Series method.
```

Comment:
Actually, for tweedie family, you perform one of the three estimations:
1. fix the variance power and estimate the dispersion parameter;
2. fix the dispersion parameter and estimate the variance power;
3. estimate both the variance power and dispersion parameter.

Please add somewhere:
When you fix the variance power and estimate the dispersion parameter, the series method is used to perform the estimation.  In this case, you can actually separate the glm coefficient estimation and the dispersion parameter estimation at the end of the glm model building process.  Standard Newton's method is used to estimate the dispersion parameter using the series which is an approximation of the tweedie likelihood function.